source/iofs/iofs_test.go

```diff
@@ -0,0 +1,22 @@
+// +build go1.16
+
+package iofs_test
+
+import (
+	"embed"
+	"testing"
+
+	"github.com/golang-migrate/migrate/v4/source/iofs"
+	st "github.com/golang-migrate/migrate/v4/source/testing"
+)
+
+func Test(t *testing.T) {
+	//go:embed testdata/migrations/*.sql
+	var fs embed.FS
```

Comment:
I'm just curious--how did this even work? 

Per the [golang docs](https://golang.org/pkg/embed/#hdr-Directives):
> The //go:embed directive can be used with both exported and unexported variables, depending on whether the package wants to make the data available to other packages. It can only be used with global variables at package scope, not with local variables.

To me this looks like a local variable, not a global variable at package scope... What am I missing?

Reply:
Local variables were still supported when implementing this PR.
https://github.com/golang/go/issues/43216
It is a remnant of that.

Reply:
Ah, thank you!